packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -299,39 +304,39 @@ interface ActionsBarProps extends IContainerActions {
 function ActionsBar(props: ActionsBarProps): React.ReactElement {
 	const { isContainerConnected, tryConnect, forceDisconnect, closeContainer } = props;
 
-	const connectButtonTooltipId = useId("connect-button-tooltip");
-	const disconnectButtonTooltipId = useId("disconnect-button-tooltip");
-	const disposeContainerButtonTooltipId = useId("dispose-container-button-tooltip");
+	// const connectButtonTooltipId = useId("connect-button-tooltip");
+	// const disconnectButtonTooltipId = useId("disconnect-button-tooltip");
+	// const disposeContainerButtonTooltipId = useId("dispose-container-button-tooltip");
 
 	const changeConnectionStateButton = isContainerConnected ? (
-		<TooltipHost content="Disconnect Container" id={disconnectButtonTooltipId}>
-			<IconButton
-				onClick={forceDisconnect}
-				disabled={forceDisconnect === undefined}
-				menuIconProps={{ iconName: "PlugDisconnected" }}
-				aria-describedby={disconnectButtonTooltipId}
-			/>
-		</TooltipHost>
+		<Button
+			icon={<PlugDisconnected24Regular />}
+			onClick={forceDisconnect}
+			disabled={forceDisconnect === undefined}
+		>
+			{" "}
+			Disconnect Container{" "}
```

Comment:
The spaces around the text feel like they shouldn't be here... if anything, do we need to add padding through css?

Reply:
No, there were trailing spaces and `prettier` somehow tried to "fix" it in odd manners. Spaces are now removed.